docs/_docs/03-ug-mutable-dataset.md

```diff
@@ -87,7 +89,83 @@ hs.refreshIndex("empIndex", "incremental")
 ```
 
 ### Optimize Index
-TODO
+The fastest way to index new data files and merge them into an existing index is by calling `refreshIndex`
```

Comment:
Maybe change "The fastest way" to "One way" since we will have "quick" mode soon? 😄